packages/frontend/pages/lp.tsx

```diff
@@ -43,42 +49,154 @@ const useStyles = makeStyles((theme) =>
     },
+    comparison: {
+      marginTop: theme.spacing(4),
+      //paddingBottom: theme.spacing(2),
+    },
+    comparisonItem: {
+      width: '400px',
+    },
+    comparisonPoint: {
+      marginTop: theme.spacing(2),
+      padding: theme.spacing(0, 2),
+    },
+    details: {
+      marginTop: theme.spacing(4),
+      // display: 'flex',
+    },
+    heading: {
+      marginTop: theme.spacing(3),
+    },
+    tradeForm: {
+      position: 'sticky',
+      top: '100px',
+    },
+    chartNav: {
+      border: `1px solid ${theme.palette.primary.main}30`,
+    },
   }),
 )
 
 export function LPCalculator() {
   const classes = useStyles()
   const { isRestricted } = useRestrictUser()
-  const { pool } = useSqueethPool()
+  const { ethPrice } = useWorldContext()
+
+  const [lpType, setLpType] = useState(0)
 
   return (
     <div>
       <Nav />
       <div className={classes.container}>
-        <div className={classes.logoContainer}>
-          <div className={classes.logo}>
-            <Image src={squeethTokenSymbol} alt="squeeth token" width={37} height={37} />
-          </div>
-          <div>
-            <Typography variant="h5" className={classes.logoTitle}>
-              Uniswap V3 LP SQTH-ETH Pool
-            </Typography>
-            <Typography className={classes.logoSubTitle} variant="body1" color="textSecondary">
-              Earn LP fees for providing SQTH-ETH liquidity
-            </Typography>
+        <div style={{ width: '800px' }}>
+          <div style={{ display: 'flex' }}>
+            <div className={classes.logo}>
+              <Image src={squeethTokenSymbol} alt="squeeth token" width={37} height={37} />
+            </div>
+            <div>
+              <Typography variant="h5" className={classes.logoTitle}>
+                Uniswap V3 LP SQTH-ETH Pool
+              </Typography>
+              <Typography className={classes.logoSubTitle} variant="body1" color="textSecondary">
+                Earn LP fees for providing SQTH-ETH liquidity
+              </Typography>
+            </div>
           </div>
           <SqueethInfo />
+          <div className={classes.details}>
+            <div style={{ display: 'flex' }}>
+              <SqueethTabs
+                style={{ background: 'transparent' }}
+                className={classes.chartNav}
+                value={lpType}
+                onChange={(evt, val) => setLpType(val)}
+                aria-label="Sub nav tabs"
+                scrollButtons="auto"
+                variant="scrollable"
+              >
+                <SqueethTab label={'Buy and LP'} />
+                <SqueethTab label="Mint and LP" />
+              </SqueethTabs>
+            </div>
+            {lpType === 0 ? (
+              <div style={{ marginTop: '16px' }}>
+                <Typography color="primary" variant="h6">
+                  Buy squeeth and LP
+                </Typography>
+                <Typography>Earn a payoff similar to ETH^1.5</Typography>
```

Comment:
```suggestion
                <Typography>Earn a payoff similar to ETH&sup1.5;</Typography>
```

Reply:
Believe this will give us the nice formatting! It's what we do for ETH^2 on trade page haha

Reply:
No it's not working that's why I didn't use it